networking module, note format, and versions (#116)
infrawatch/documentation: doc-Service-Telemetry-Framework/modules/proc_deploying-with-routed-l3-networks.adoc

//
// Start the title with a verb, such as Creating or Create. See also
// _Wording of headings_ in _The IBM Style Guide_.

> [id="deploying-to-non-standard network-topologies_{context}"]

Reviewer: ```suggestion
[id="deploying-to-non-standard-network-topologies_{context}"]
```

= Deploying to non-standard network topologies

> If your nodes are on a separate network from the default `InternalApi` network, you must make several changes so that {MessageBus} can transport data to the {Project} ({ProjectShort}) server instance. This scenario is typical in a spine-leaf or a DCN topology. For more information about DCN configuration, see the https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/{vernum}/html-single/spine_leaf_networking/[Spine Leaf Networking] guide.

Reviewer: ```suggestion
If your nodes are on a separate network from the default `InternalApi` network, you must make configuration adjustments so that {MessageBus} can transport data to the {Project} ({ProjectShort}) server instance. This scenario is typical in a spine-leaf or a DCN topology. For more information about DCN configuration, see the https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/{vernum}/html-single/spine_leaf_networking/[Spine Leaf Networking] guide.
```

// TODO: remove this after OSP13 z13 since it will no longer be necessary.

> If you use {ProjectShort}  with {OpenStack} {OpenStackVersion}  and plan to monitor your Ceph, Block, or Object storage nodes, you must make a similar adjustment to what is done for spine-leaf and DCN network configuration. To monitor Ceph nodes, use the `CephStorageExtraConfig` parameter to define which network interface should be loaded into the QDR and collectd configuration files.

Reviewer: ```suggestion
If you use {ProjectShort} with {OpenStack} {OpenStackVersion}  and plan to monitor your Ceph, Block, or Object storage nodes, you must make a similar adjustment to what is done for spine-leaf and DCN network configuration. To monitor Ceph nodes, use the `CephStorageExtraConfig` parameter to define which network interface should be loaded into the QDR and collectd configuration files.
```
